src/components/impl/gumlet-image.js

```diff
@@ -36,10 +37,19 @@ export function GumletImage(props) {
 
   useEffect(loadGumlet);
 
+  useEffect(()=> {
+    imageEl.current.addEventListener('error',()=> {
+      if(publisherData.fallbackImage)
+       imageEl.current.src=  publisherData.fallbackImage
+    })
+  },[])
+
+  
+
   return <React.Fragment>
-    <Tag {...imageProps} {...omit(props, USED_PARAMS)} className={className ? `qt-image ${className}` : 'qt-image'} />
+    <Tag ref={imageEl} {...imageProps} {...omit(props, USED_PARAMS)} className={className ? `qt-image ${className}` : 'qt-image'} />
     <noscript>
-      <img src={`https://${imageCDN}/${image.path(aspectRatio, {...imgParams, w: 1200})}`} alt={props.alt || ""} />
+      <img   src={`https://${imageCDN}/${image.path(aspectRatio, {...imgParams, w: 1200})}`} alt={props.alt || ""} />
     </noscript>
```

Comment:
will the Tag throw error or `img` tag itself?

Reply:
actually only when Tag==img . I should add case only for that.